openssl/src/x509/mod.rs

```diff
@@ -64,6 +64,34 @@ foreign_type! {
     pub struct X509StoreContextRef;
 }
 
+pub struct X509StoreContextBuilder(X509StoreContext);
+
+impl X509StoreContext {
+    pub fn builder() -> Result<X509StoreContextBuilder, ErrorStack> {
+        unsafe {
+            ffi::init();
+            cvt_p(ffi::X509_STORE_CTX_new()).map(|p| X509StoreContextBuilder(X509StoreContext(p)))
+        }
+    }
+
+    #[cfg(any(all(feature = "v102", ossl102), all(feature = "v110", ossl110)))]
```

Comment:
Oh... it might in fact be. I thought I couldn't find it in 1.0.1 initially, double checking it looks like it is.